### PR TITLE
upgrade elasticsearch version to 6.8.3

### PR DESCRIPTION
**Pull request by nrzhao4**
Upgrading elasticsearch to version 6.8.3

### CODE REVIEW

**Review by coverbeck — APPROVED** [review]
Approved, but please don't merge until you can merge your web service and UI PRs at the same time, so we don't break dev.

**Review by denis-yuen — APPROVED** [review]
Ditto

**Pull request merged**
